src/functions/acalc.h

```diff
@@ -2,6 +2,7 @@
 #define ACALC_H
 
 #include <QDateTime>
+#define _USE_MATH_DEFINES // needed for MSVC
```

Comment:
If only one compiler needs this it should be in a guard
https://stackoverflow.com/questions/5850358/is-there-a-preprocessor-define-that-is-defined-if-the-compiler-is-msvc
```c++
#if _MSC_VER
...
#endif

Reply:
fixed

Reply:
*CORRECTION*
```c++
#ifdef _MSC_VER
...
#endif
```